src/components/MLTaskManager.js

```diff
@@ -26,14 +24,22 @@ class MLTaskManager extends React.Component {
         
         <fieldset>
           <legend>Status</legend>
-          <var>{mlTask.status}</var>
+          <var>Task: {mlTask.status} / Results: {mlResults.status}</var>
         </fieldset>
         
         <fieldset>
-          <legend>URL</legend>
+          <legend>ML Task Request ID</legend>
           <div className="flex-row">
-            <input className="text input flex-item grow" value={state.url} onChange={(e) => { this.setState({ url: e.target.value }) }} />
-            <button className="action button flex-item" onClick={() => { mlTask.testFetch(state.url) }}>Fetch</button>
+            <input className="text input flex-item grow" value={mlTask.id} onChange={(e) => { mlTask.setId(e.target.value) }} />
```

Comment:
Maybe list the input tag's attributes vertically here? The long line can be hard to read.

Reply:
👍